Preliminar 3 pages

ramondiaz21 opened the pull request:
3 preliminar pages
- New account
- Check email
- Restart password
no responsive yet

ramondiaz21 committed:
Preliminar 3

3 preliminar pages

ramondiaz21 committed:
Login

Login screen, lack to do it responsive and narvbar/footer

ramondiaz21 committed:
Contact

Contact page, lack to do it responsive, navbar and footer

AlanGtz commented on check_mail.html:
```diff
@@ -0,0 +1,31 @@
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+<body class="check-body">
+		<section class="check-section">
```
You have to change your indentation in all html files, code should always look clean and organized. To achieve this, leave just two spaces of indentation for each nested level, there's no need of so much space to notice one element is inside another.

AlanGtz commented on check_mail.html:
```diff
@@ -0,0 +1,31 @@
+<body class="check-body">
+		<section class="check-section">
+			<div class="check-div">
+				<form class="check-form">
```
I believe this is not a form, there's no inputs, no selects, nor submit button, is just an advice, so there's no need to use a form, a simple div could work.

AlanGtz commented on check_mail.html:
```diff
@@ -0,0 +1,31 @@
+						carpetas de correo no deseado, </br>
+						sociales u otras.</p>
+
+						<p class="check-p1"><strong>No he recibido el correo electrónico</strong></p>
```
Shouldn't this be a button?

AlanGtz commented on css/main.css:
```diff
@@ -1047,6 +1047,354 @@ a {
 .active3 {
   background-image: url("../assets/images/sliderHero3.jpg"); }
 
+.restart-body {
```
Are you using Koala for preprocessing sass? There's an option to compile compressed files, use it, the file will be lighter and this is not readable anyways.

AlanGtz commented on sass/main.scss:
```diff
@@ -24,6 +24,12 @@
   'pages/blog/blog',
   'pages/blog/post';
 
+
```
-You're using the pattern 7-1, well done :+1:

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+.check-body{
+	background-color: #F7F7F7;
+
+	.check-section{
```
As in the html, you need only two spaces of indentation.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+.check-body{
+	background-color: #F7F7F7;
```
Use variables instead of color codes.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+						font-weight: 300;
+						margin-bottom: 50px;
+						margin-top: 50px;
+					}
```
No more than 3 levels of nesting, make it readable.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+		background-color: $white;
+		text-align: center;
+
+		.check-div{
```
The correct syntax is:
.classname {
  space ---- :arrow_up:  here
}

You have the sass guide for reference, the cleaner the better.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+
+		.check-div{
+			width: 30%;
+			@include center(true,true);
```
Do not use this mixin unless it's truly necessary, find another approach for this.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+				.check-field{
+					border-radius: 7.5px;
+					border: none;
+					-webkit-box-shadow: 0px 0px 10px 2px rgba(184,175,184,0.51);
```
If you're using Koala, you don't need to add every prefix, just activate autoprefixer checkbox.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+
+					.check-h1{
+						height: 5.14%;
+						color: #757779;
```
Remember, variables.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+					}
+					
+					.check-p{
+						height: 5.14%;
```
Why is this height a percentage? Feels weird to me.

AlanGtz commented on sass/pages/check-mail/_check_mail.scss:
```diff
@@ -0,0 +1,56 @@
+						font-family: Muli-Light;
+						font-size: 15px;
+						font-weight: 300;
+						margin-bottom: 50px;
```
margin: 50px 0;    would have the same effect as including margin-top and margin-bottom separately. Read about properties syntax.
You could set the four values like:
- margin/padding: top  right  down  left;

In that order.

Or set just two and it will read as:
- margin/padding: top&bottom  left&right;

Even if you write three values will work if you need:
- margin/padding:  top  left&right  bottom;

AlanGtz commented on sass/pages/contact/_contact.scss:
```diff
@@ -0,0 +1,89 @@
+			display: block;
+		}
+
+		.contact-p1{
```
What does "p1" mean? If I'm taking a look at your code I'll never know what this class is for.

AlanGtz commented on sass/pages/login/_login.scss:
```diff
@@ -0,0 +1,108 @@
+}
+
+section{
+	width: 100vh;
```
You don't need to specify the height of the section, it should grow to fit the size of its inner elements, this is why you don't have to use the center mixin, when an element is absolute positioned the height disappears.

AlanGtz commented on sass/pages/login/_login.scss:
```diff
@@ -0,0 +1,108 @@
+	background-color: #F7F7F7;
+
+	.login-div{
+		width: 25%;
```
are you sure this always works? Did you try with mobile sizes?

AlanGtz commented on sass/pages/new-account/_new_account.scss:
```diff
@@ -0,0 +1,90 @@
+			margin: 0;
+		}
+
+		.new-inputs{
```
I don't understand why you always use so many properties, I'm not sure you need many of them, take a better look at your code, try to use just the necessary. For example, if you define a font-family in the parent, all its children will use the same font, try to have things like that in mind.

AlanGtz commented on sass/pages/restart-pass/_restart_pass.scss:
```diff
@@ -0,0 +1,59 @@
+		
+		.restart-div{
+			width:25%;
+			@include center(true,true);
```
Try not to mix mixins with the other properties, make it readable: mixins first, then a line break, then the rest of properties.

AlanGtz commented on sass/pages/restart-pass/_restart_pass.scss:
```diff
@@ -0,0 +1,59 @@
+					.restart-text{
+						font-family: Muli-ExtraLight;
+						background-color: $white;
+						border-width: 0;
```
You really need this property?

AlanGtz reviewed (changes requested):
There's a lot of fixes to work on, if you have a reason to do something that I don't understand, just leave a comment explaining. Don't worry too much, this is just the first pull request, you're learning fast and that's a good thing

ramondiaz21 committed:
Check_mail

corrections in check_mail.html and the scss, lack to do it the responsive, navbar and footer.

ramondiaz21 committed:
variables

change codes for variables of colors

ramondiaz21 committed:
Corrections

All corrections are made, except the scss identation, for the next time ill get better, promise! just lack to do it responsive and the integration of navbar and footer

ramondiaz21 committed:
Footer added

just stack to integrate the navbar and make it responsive

ramondiaz21 committed:
update

just and main.css update

ramondiaz21 committed:
last update

I add the navbar and some other things to the forms, it still lack to do it responsive, im working on it
